task06.2/task06.2/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace StringConcatTask
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите слово перевозка, чтобы получить новые слова");
+            string perevozka = Console.ReadLine();
+
```

Comment:
Не вижу смысла вводить слово с клавиатуры
```suggestion
            string perevozka = "перевозка";

```